src/parser/demonhunter/havoc/modules/talents/Felblade.js

```diff
@@ -41,13 +42,18 @@ class Felblade extends Analyzer {
       return;
     }
     this.addEventListener(Events.energize.by(SELECTED_PLAYER).spell(SPELLS.FELBLADE_PAIN_GENERATION), this.onEnergizeEvent);
+    this.addEventListener(Events.damage.by(SELECTED_PLAYER).spell(SPELLS.FELBLADE_PAIN_GENERATION), this.onDamageEvent);
   }
 
   onEnergizeEvent(event) {
     this.furyGain += event.resourceChange;
     this.furyWaste += event.waste;
   }
 
+  onDamageEvent(event) {
+    this.damage += event.amount;
+  }
```

Comment:
damage events should include absorbed damage. ``this.damage += event.amount + (event.absorbed || 0);``

Reply:
Fixed